pyflakes/checker.py

```diff
@@ -1973,7 +1988,20 @@ def DICT(self, node):
     def IF(self, node):
         if isinstance(node.test, ast.Tuple) and node.test.elts != []:
             self.report(messages.IfTuple, node)
-        self.handleChildren(node)
+
+        prev = self._in_type_checking
+        if _is_typing(node.test, 'TYPE_CHECKING', self.scopeStack):
+            self._in_type_checking = True
+        # Handle the body of this if statement.
+        body_nodes = node.body
+        if not isinstance(body_nodes, list):
+            body_nodes = [body_nodes]
```

Comment:
does this code ever run? I think the body is always a list here (?)

Reply:
in `IFEXPR` body is not a list.

Reply:
Yes, there is a test that will fail if this if statement is removed.